Week 06/Lecture 11/Assignment 01/employee/src/main/java/com/example/employee/specification/EmployeeSpecification.java

```diff
@@ -0,0 +1,89 @@
+package com.example.employee.specification;
+
+import com.example.employee.common.Gender;
+import com.example.employee.criteria.EmployeeSearchCriteria;
+import com.example.employee.entity.Department;
+import com.example.employee.entity.DeptEmp;
+import com.example.employee.entity.Employee;
+import com.example.employee.entity.Title;
+import jakarta.persistence.criteria.*;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.util.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeSpecification implements Specification<Employee> {
```

Comment:
This should be in repository package

Reply:
Thank you for the feedback, I've move the EmployeeSpecification to the repository package in my latest commit.